.clang-tidy

```diff
@@ -6,6 +6,6 @@ Checks: >
     modernize-use-override
 CheckOptions:
     - key: cppcoreguidelines-macro-usage.AllowedRegexp
-      value: 'validate_global_widx'
+      value: 'validate_global_widx|NETWORK_STREAM_VERSION'
```

Comment:
just make network stream a constexpr

Reply:
This is not terribly easy as the value is used in another define.

Reply:
As @Gymnasiast said its used to generate the version macro so I felt like this is the better approach for the time being.